### ISSUE TITLE
Add `prefers-reduced-motion` media query option for animations

### ISSUE THREAD

**Issue by LZoog** [mozilla/fxa#2289]
First noticed while looking into #1994.

Since our animations are often set to `infinite` (over the ADA recommended `5s` limit), we should allow for users to opt-in to [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).

--

Ideally, we should be able to add a class (such as `animate`) to all animated SVGs and elements so that we can have one sweeping media query:
```
@media (prefers-reduced-motion: reduce) {
  .animate {
    animation: none;
  }
}
```

However, it may not be quite this simple because we set our animated SVG styles/animations inline and then use them as `background-image`s. The less preferred alternative would be to add this media query to each SVG.
